README.md

```diff
@@ -6,59 +6,119 @@ DUS leverages the power of Amazon Textract, Amazon Comprehend , Amazon Comprehen
 
 ## Architecture Diagram
 
-
 ![img](./images/DUS_Arch.png)
 
+Current document formats supported : **PDF,JPG,PNG**
 
+## 2. CICD Deploy
 
-Current document formats supported : **PDF,JPG,PNG**
+### Requirements
 
+- aws cli
 
-##  Development Deploy
+  `sudo yum -y install aws-cli`
 
-The instructions below cover installation on Unix-based Operating systems like macOS and Linux. 
-You can use a AWS Cloud9 enviornment or EC2 instance (recommended: t3.large or higher on Amazon Linux platform) to deploy the solution
+- pip3 (Required to install packages)
+
+  `curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py`
+
+### Getting Started with Full Deploy
+
+- Create a bucket to act as the target Amazon S3 distribution bucket
+
+_Note:_ You will have to create an S3 bucket with the template 'my-bucket-name-<aws_region>'; aws_region is where you are testing the customized solution.
+
+For example, you create a bucket called `my-solutions-bucket-us-east-1`,
+
+- Now build the distributable:
+
+```
+chmod +x ./deployment/build-s3-dist.sh
+./deployment/build-s3-dist.sh <bucket-name-minus-region> <solution-name> <version>
+```
+
+For example,
+
+```
+./deployment/build-s3-dist.sh my-solutions-bucket document-understanding-solution v1.0.0
+```
+
+- Deploy the distributable to an Amazon S3 bucket in your account. _Note:_ you must have the AWS Command Line Interface installed.
+
+```
+aws s3 cp ./deployment/global-s3-assets/ s3://my-bucket-name-<aws_region>/<solution_name>/<my-version>/ --recursive --acl bucket-owner-full-control --profile aws-cred-profile-name
+aws s3 cp ./deployment/regional-s3-assets/ s3://my-bucket-name-<aws_region>/<solution_name>/<my-version>/ --recursive --acl bucket-owner-full-control --profile aws-cred-profile-name
+```
+
+- Get the link of the document-understanding-solution.template uploaded to your Amazon S3 bucket.
+- Deploy the Document Understanding solution to your account by launching a new AWS CloudFormation stack using the link of the document-understanding-solution.template.
+
+```
+aws cloudformation create-stack --stack-name DocumentUnderstandingSolutionCICD --template-url https://my-bucket-name-<aws_region>.s3.amazonaws.com/<solution_name>/<my_version>/document-understanding-solution.template --parameters ParameterKey=Email,ParameterValue=<my_email> --capabilities CAPABILITY_NAMED_IAM --disable-rollback
+```
+
+This solutions will create 7 S3 buckets that need to be manually deleted when the stack is destroyed (Cloudformation will only delete the solution specific CDK toolkit bucket. The rest are preserved to prevent accidental data loss).
+
+- 2 for CICD
```

Comment:
We have a few other buckets being created:

1. Covid -19 bucket
2. Bulk processing bucket
More info on these here: https://github.com/awslabs/document-understanding-solution/blame/kendraMaster/README.md#L80

Reply:
updated